plugins/ltac/pptactic.ml

```diff
@@ -1300,11 +1300,11 @@ let register_basic_print0 wit f g h =
 
 
 let pr_glob_constr_pptac c =
-  let _, env = Pfedit.get_current_context () in
+  let _, env = Pfedit.get_current_context () in (* XXX *)
```

Comment:
Here again I'm not sure I understand the logic. Isn't the glob_constr printer going to do things differently (in terms of disambiguating names) from the raw printer? Is this really desired? @ppedrot

Reply:
No idea, but see my comment above.

Reply:
Indeed you are asking a deep question on the design of printing; the current situation indeed has this problem, but it must be said that printing from elaborated terms is a quite hard problem.

None of the solutions I know are easy, what I do in some other compilers is to actually resolve names to some extent at parsing time, but yeah, this is harder here with proofs....